### DISCUSSION TITLE
럭키비키: 부정적인 사고를 초긍정적으로 바꿔주는 앱🍀

### DISCUSSION THREAD

**Discussion by namudongs** [DeveloperAcademy-POSTECH/.github#66] [Show and tell]
안녕하세요, 3기 오후반 나무디입니다. 여러분의 일상을 초긍정으로 물들일 혁신적인 앱, '럭키비키'를 소개합니다!

'럭키비키'는 부정적인 생각을 긍정적으로 바꿔주는 신개념 앱으로, 걸그룹 아이브(IVE)의 장원영에서 영감을 받아 탄생했습니다. 
이 앱은 사용자가 입력한 부정적인 문장을 '원영적 사고'를 바탕으로 한 초긍정적인 메시지로 변환해줍니다.

![럭키비키](https://github.com/user-attachments/assets/cf39901c-ca04-45ce-b13b-b14defd7ac37)

## 주요 특징
1. 부정문 변환: "오늘 비가 와서 짜증나" → "오늘 비가 와서 공기가 맑아졌어, 완전 럭키비키잖아!"
2. 스트레스 감소 및 행복감 증진
3. 긍정적 에너지 확산

## 개발 스토리
'럭키비키'는 주말에 심심했던 제가 단 이틀 만에 기획부터 출시까지 완료된 열정 가득한 앱입니다.
이틀 동안 기획-디자인-개발-출시까지의 플로우를 경험하며 얻고 배운 게 정말 많았어요!
출시 한달만에 270명의 유저를 확보했습니다.

## 기술 스택
- Firebase Authentication
- Firebase Firestore
- ChatGPT API

## 향후 계획
- 커뮤니티 & 히스토리 기능
- 긍정일기 (일기를 작성하면, 여러 사고로 변환해서 댓글을 달아줍니다)
- 다양한 '~적 사고' 변환 옵션 (흥민적, 명수적, 홍철적 사고 등)
---
'럭키비키'와 함께라면 모든 순간이 행운으로 가득할 거예요! 지금 바로 [다운로드](https://apps.apple.com/kr/app/%EB%9F%AD%ED%82%A4%EB%B9%84%ED%82%A4/id6520379298)하여 여러분의 일상을 초긍정으로 물들여보세요. 
이거 완전 럭키비키잔앙(◠‿◠)**🍀

**Comment by 00yhsp** [comment]
2024 최고의 앱
